src/lib/utils/getRefinements.ts

```diff
@@ -85,66 +84,89 @@ function getRefinements(
   clearsQuery: boolean = false
 ): Refinement[] {
   const res: Refinement[] = [];
-
-  forEach(state.facetsRefinements, (refinements, attributeName) => {
-    forEach(refinements, name => {
+  const {
+    facetsRefinements = {},
```

Comment:
just checking, when these are not set, it's undefined right (not `null`)?

Reply:
From the helper typings coming from DefinitelyTyped, these can be `undefined` (not `null`).

From my experiments, these are set by default in the search parameters (accordingly to `{}` or `[]`). I'm not sure in which case they can be `undefined` in the real world. I figured I would replicate almost exactly what Lodash did.